crowbar_framework/app/helpers/nodes_helper.rb

```diff
@@ -20,48 +20,28 @@ def nodes_by_role(role)
     NodeObject.find("roles:#{role}").sort_by(&:alias)
   end
 
-  def piechart_for(group)
-    values = piechart_values(group)
-    tooltip = piechart_tooltip(values)
-
-    content_tag(
-      :span,
-      "",
-      :title => tooltip,
-      "data-piechart" => values.join(","),
-      "data-tooltip" => "true"
+  def link_to_node(node, handle)
+    link_to(
+      node_icon(node) + " " + truncate(node[:alias], :length => 25),
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)